esx_service/utils/auth_api.py

```diff
@@ -491,21 +491,43 @@ def check_datastore(datastore_name):
     if not datastore_exist(datastore_name):
         error_info = error_code.generate_error_info(ErrorCode.DS_NOT_EXIST, datastore_name)
         return error_info
-    
+
     return None
 
 def privilege_exist(privileges, datastore_name):
-    """ Check wheter a entry with given datastore_name exists in privileges """
+    """ Check whether a entry with given datastore_name exists in privileges """
     for p in privileges:
         ds_name = vmdk_utils.get_datastore_name(p.datastore_url)
         if ds_name == datastore_name:
             return True
-    
+
     return False
-    
+
+def check_privilege_parameters(privilege):
+    """ Check whether the privilege parameters are invalid """
```

Comment:
Add some description of the input param "privilege" and also the description of the return value of this function.